qt/scientific_interfaces/Inelastic/Analysis/IndirectFitDataView.h

```diff
@@ -49,6 +46,7 @@ public slots:
   void setCell(std::unique_ptr<QTableWidgetItem> cell, size_t row, size_t column);
 
 private:
+  QStringList HeaderLabels;
```

Comment:
Sorry to be nitpicky! We tend to prefix member variables with `m_`  https://developer.mantidproject.org/Standards/CPPStandards.html#naming-conventions

For example:
https://github.com/mantidproject/mantid/blob/51c7731b65bae4a20f9baf88dae59cba8518b1f5/qt/widgets/common/inc/MantidQtWidgets/Common/Hint.h#L21

Reply:
No, it's ok. Sorry for giving you extra work. 
I though that m_ prefix applied only to public members. I'll fix it.